### PR TITLE
events

### PR DESCRIPTION
**Pull request by GrishinN**

### CODE REVIEW

File: UI/dom.js [hunk 1]
```diff
@@ -62,14 +71,41 @@ let domFunction = (function () {
                 let photoLink = photoLink_mas[0].firstChild;
                 photoLink.src = photoPost.photoLink;
             }
+            if('hashtags' in photoPost){
+                let hashtags_mas = post.getElementsByClassName('hashtag');
```
Comment (tyndria) [inline UI/dom.js:75]: https://www.w3schools.com/js/js_conventions.asp
We use camelCase for Javascript, no underscores

**Comment by tyndria** [comment]
there are some problems with this dropdown
screen shot 2018-05-28 at 17 42 50

File: UI/dom.js (hunk 1 above)
Comment (GrishinN) [reply UI/dom.js:75]: @tyndria , исправил.

**Comment by GrishinN** [comment]
@tyndria ,пофиксил .
default

**Pull request merged**